dogs-initial/src/main/java/com/lambdaschool/dogsinitial/controller/DogController.java

```diff
@@ -17,13 +22,20 @@
 @RequestMapping("/dogs")
 public class DogController
 {
+    private static final Logger logger= LoggerFactory.getLogger(DogController.class);
+
+//    @Autowired
+//   RabbitTemplate rt;
 
 
     // localhost:2020/dogs/dogs
     @GetMapping(value = "/dogs",
                 produces = {"application/json"})
     public ResponseEntity<?> getAllDogs()
     {
+        logger.info("/dogs/dogs accessed");
+//         MessageDetail message = new MessageDetail("/dogs/dogs accessed", 7, false);
+//        rt.convertAndSend(DogsinitialApplication.QUEUE_NAME_HIGH,message);
         return new ResponseEntity<>(DogsinitialApplication.ourDogList.dogList, HttpStatus.OK);
```

Comment:
You can add the date/time of access to your log/message text by concatenating `new Date()` to your log string (ex.: `new Date() + " endpoint accessed."`).

Also, why comment out the lines for sending to your create message queue? It would actually work properly.